config/config_files/codechecker.json

```diff
@@ -4,6 +4,10 @@
     "--enable=core.CallAndMessage",
     "--analyzer-config",
     "clangsa:unroll-loops=true",
+    "--analyzer-config",
```

Comment:
isn't this config file just an example? Will it get in effect by default?

Reply:
I initially intended to put the change here, I'm just not sure whether hardcoding such a change is appropriate:
https://github.com/Ericsson/codechecker/blob/master/analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py#L392
@bruntib?